packages/pf3-component-mapper/src/form-fields/plain-text.js

```diff
@@ -1,11 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const PlainText = ({ label, name }) => label.split('\n').map((paragraph, index) => <p key={ `${name}-${index}` }>{ paragraph }</p>);
+const PlainText = ({ variant, label, name }) => label.split('\n').map((paragraph, index) =>
+  React.createElement(`${variant}`, { key: `${name}-${index}` }, paragraph));
 
 PlainText.propTypes = {
+  variant: PropTypes.string,
```

Comment:
Maybe we should add a list of supported strings. I would do:
```jsx
variant: PropTypes.oneOf(['p', 'span', 'h1' ... 'h6', 'div', ...])
```

Maybe do some research for valid text elements?

Reply:
Okey :). I initially went with the "you can destroy your form whatever way you want" path. I'll add them.